### ISSUE TITLE
[BUG-1.16.5] Server crashes when world generates for the first time. (Gets stuck at preparing spawn area X%.)

### ISSUE THREAD

**Issue by Gbergz** [How-Bout-No/Outvoted#109]
**Describe the bug**
Server crashes when world generates for the first time. 
(Gets stuck at preparing spawn area X%.)

**To Reproduce**
1. Go to install mod on server.
2. Start it.
3. Let it try to generate world.
4. Stuck.
5. Crash but doesn't close, gets stuck at preparing spawn area.

**Logs**
https://gist.github.com/Gbergz/6081a1fea409ae759e2d8e36bd69d1f3

**Version (please complete the following information):**
 - Minecraft Version : 1.16.5
 - Modloader: Forge
 - Mod Version: 2.0.0-alpha.12 | https://www.curseforge.com/minecraft/mc-mods/outvoted/files/3455191

**Comment by Gbergz** [comment]
Ignore this. Regenerated the config and now it seems to work fine.

**Comment by Gbergz** [comment]
Seems to happen again after some restarts. How weird..

```
Caused by: java.lang.NoSuchMethodError: net.minecraft.world.spawner.WorldEntitySpawner$EntityDensityManager.func_234988_a_()I
        at io.github.how_bout_no.outvoted.world.SpawnUtil.isBelowCap(SpawnUtil.java:14) ~[outvoted:?] {re:classloading}
        at io.github.how_bout_no.outvoted.world.feature.BurrowGenFeature.generate(BurrowGenFeature.java:38) ~[outvoted:?] {re:classloading}
```

**Comment by Gbergz** [comment]
Any clues?

**Comment by How-Bout-No** [comment]
Interesting. To take it at face value a method is missing that shouldn't be missing as far as I am aware. Even more odd is that it fixed itself after a *config* reset? I'll take a look if I have time aside from 1.5.

**Comment by Gbergz** [comment]
> Even more odd is that it fixed itself after a config reset?

Seems it was just a fluke, it's like 20% success on boot now. (that it actually starts).

**Comment by How-Bout-No** [comment]
> Seems it was just a fluke, it's like 20% success on boot now. (that it actually starts).

Is this on the same server? And are you using a certain world seed or generating randomly? I haven't been able to recreate this on my end so far.

**Comment by Gbergz** [comment]
Same server. Crash happens almost every time.
Same logs as above.

**Comment by How-Bout-No** [comment]
Yea I can't manage to replicate the error on my end, at least with just Outvoted and the dependencies. Server always starts fine without issue. Are you using the latest Forge version?

**Comment by Gbergz** [comment]
Hm, strange. Yes latest forge.

**Comment by How-Bout-No** [comment]
I'll modify/mess around with my test env to see if I can get this error to come out. I think I might just need to force a spawn near the desert.

**Comment by How-Bout-No** [comment]
Ah, wait, now I recognize the error. Been a while since I've worked on 2.0 but this is the same issue as #82 and #83, I just never published a fix for 1.16.5 :P
Kinda overcomplicated this, but safe to say I was able to reproduce the crash and have a fix ready.

**Comment by How-Bout-No** [comment]
Fixed in alpha 13
